rotkehlchen/db/history_events.py

```diff
@@ -198,9 +198,23 @@ def delete_history_events_by_identifier(
 
         If any identifier is missing the entire call fails and an error message
         is returned. Otherwise None is returned.
+
+        For asset movements the fee and the main event are linked. Deleting one of them
```

Comment:
Why do you want to add this logic? I would say it's a can of worms. Don't do this. As you can see adds weird conditionals everywhere and does not scale.

Do we delete the entire eth staking event if wejust delete the mev event?
Do we delete the entire swap if we just remove the send?
Do we delete the entire  safe transaction if we just delete the info event of safe execution completion?

etc.

I guess you get the problem. This approach does not scale. Handle this in the frontend, or in the api conditional where  asset movements are deleted, to alway delete both.

Please undo this